webpack.config.js

```diff
@@ -373,27 +373,38 @@ let appConfig = {
       ),
     },
 
+    fallback: {
+      vm: require.resolve('vm-browserify'),
+      stream: require.resolve('stream-browserify'),
+      crypto: require.resolve('crypto-browserify'),
```

Comment:
Need to wait for a new `eslint-import-resolver-webpack` release in order to get rid of these (I think): https://github.com/benmosher/eslint-plugin-import/issues/631